accountmodel.cpp

```diff
@@ -2,10 +2,11 @@
 
 AccountModel::AccountModel() {}
 
-AccountModel::AccountModel(int id, QString  login, QString password, bool isActive)
+AccountModel::AccountModel(int id, QString  login, QString password, bool isActive, QString rule)
```

Comment:
Именование не соответствует именованию в БД.
rule - правило
role - роль
```suggestion
AccountModel::AccountModel(int id, QString  login, QString password, bool isActive, QString role)
```